include/zephyr/display/cfb.h

```diff
@@ -51,8 +52,72 @@ struct cfb_font {
 };
 
 struct cfb_position {
-	uint16_t x;
-	uint16_t y;
+	int16_t x;
+	int16_t y;
+};
+
```

Comment:
For private structures, it would be best to hide them from Doxygen. This requires an `@endcond` after the hidden declarations.

```suggestion
/**
 * @cond INTERNAL_HIDDEN
 */
```

Reply:
Thank you for your review.
I fixed the pointing out.
And I also cleaned up the overall doxygen output.